controllers/azuresqluser_controller.go

```diff
@@ -30,7 +30,7 @@ type AzureSQLUserReconciler struct {
 }
 
 // +kubebuilder:rbac:groups=azure.microsoft.com,resources=AzureSQLUsers,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=azure.microsoft.com,resources=AzureSQLUsers/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=azure.microsoft.com,resources={AzureSQLUsers/status,AzureSQLUsers/finalizers},verbs=get;update;patch
```

Comment:
The casing on this is strange.

Nothing you can do about it I know... just observing =)